pkg/resources/testdata/TestAcc_GrantPrivilegesToDatabaseRole_AlwaysApply/test.tf

```diff
@@ -0,0 +1,6 @@
+resource "snowflake_grant_privileges_to_database_role" "test" {
+  database_role_name = "\"${var.database}\".\"${var.name}\""
+  all_privileges     = var.all_privileges
+  on_database        = "\"${var.database}\""
```

Comment:
in every example this is with `\"`. Is it possible to just use unquoted database or not? I've checked description but I don't think it's right.

Reply:
you should be able to, but I did it with quotes because then it's easier to compare identifiers in acceptance test

Reply:
I just checked that it didn't matter and works as intended, so I'll leave it without quotes in this test for variety